go.mod

```diff
@@ -1,8 +1,10 @@
-module github.com/leekchan/accounting
+module github.com/djackreuter/accounting
```

Comment:
I suppose that you don't need to modify this line also on upstream? :)

Reply:
ahh yes oops :)